src/renderers/webgl/WebGLUniforms.js

```diff
@@ -597,6 +613,50 @@ function setValueT1Array( gl, v, textures ) {
 
 }
 
+function setValueT2DArray1Array( gl, v, textures ) {
+
+	var cache = this.cache;
+	var n = v.length;
+
+	var units = allocTexUnits( textures, n );
+
+	if ( arraysEqual( cache, units ) === false ) {
```

Comment:
For performance reasons, array uniforms are not cached anymore, see #16366

Reply:
Fixed by removing the uniform setter that shouldn't have been part of this PR anyway.